lxd/api_cluster.go

```diff
@@ -47,6 +47,9 @@ import (
 	"github.com/lxc/lxd/shared/version"
 )
 
+type evacuateStopFunc func(instance.Instance) error
+type evacuateMigrateFunc func(*Daemon, *http.Request, instance.Instance, *db.NodeInfo, bool, map[string]any, *operations.Operation) error
```

Comment:
Adding the argument names here would make it easier for people to implement this function as its very hard to understand what `bool` and `map[string]any` represent from only the function signature.